README.md

```diff
@@ -1 +1,256 @@
-# capstone-project---Conference-page
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+
+
+  <h1><b> Portoflio Website Mobile Set</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖  Capstone conference Page Project <a name="about-project"></a>
+
+
+This is a conference page designed to showcase my experiences. The website is built using HTML and CSS. Also for code uniformity. Microverse linters have been incorporated into the code
+
+## 🛠 Built With <a name="built-with"></a>
+<ul>
+  <li>HTML</li> 
+  <li>CSS</li>
+  <li>Javascript</li>
+</ul>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Mark up Language</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Styling</summary>
+  <ul>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Repository</summary>
+  <ul>
+    <li><a href="https://www.https://github.com//">Github</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- It is responsive
+- CSS, HTML and Javascript linters have been incorporated
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [PortFolio](https://santosh-konappanavar.github.io/PortFolio/)
+
+- Click here to see [loom video](https://www.loom.com/share/3174f93cb3414ca390828e69afc0a749)
```

Comment:
- [ ]  Please update these information with the capstone project because now all link are for the Portfolio.


- When clicking on your deployed link I am not able to see your page and I think it would be better to fix that, please try to update the `home.html` to `index.html` because the githubpages will search for the file to deploy it .


For your video please respect the above requirements as mentioned 👇 

![image](https://user-images.githubusercontent.com/75126481/219553882-e2fb7a9c-df85-44a4-91d1-bac017d5559e.png)